src/components/Notification/icons/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import InfoIcon from './infoIcon';
+import ErrorIcon from './errorIcon';
+import SuccessIcon from './successIcon';
+import WarningIcon from './warningIcon';
+import '../styles.css';
+
+const iconMap = {
+    info: () => <InfoIcon />,
+    error: () => <ErrorIcon />,
+    warning: () => <WarningIcon />,
+    success: () => <SuccessIcon />,
+};
+
+export default function Icon(props) {
+    const {
+        className,
+        style,
```

Comment:
This component does not receive a className and a style prop. Remember this props are only passed to the container component.